docker-compose.yml

```diff
@@ -63,10 +63,14 @@ services:
       context: ./infrastructure/containers/postgres/automatic-backup/
       dockerfile: Dockerfile
     environment:
+      BACKUP_BUCKET: rootski-database-backups
       BACKUP_DIR: /backups
       BACKUP_INTERVAL: 1h
+      # AWS connection variables
+      BACKUP_DB__AWS_ACCESS_KEY_ID: ${BACKUP_DB__AWS_ACCESS_KEY_ID?Variable not set}
+      BACKUP_DB__AWS_SECRET_ACCESS_KEY: ${BACKUP_DB__AWS_SECRET_ACCESS_KEY?Variable not set}
       # database connection variables
-      POSTGRES_HOST: postgres
+      POSTGRES_HOST: host.docker.internal
```

Comment:
Fun gotcha: `host.docker.internal` only works if your running docker containers on a Mac. If you're running docker via the WSL or on native Linux, you actually have to use `localhost` or `127.0.0.1`. 

That said, you _could_ replace `host.docker.internal` here with `postgres` since the name of the service running postgres is called `postgres` in the `services:` section of this `docker-compose.yml` file. 

However! That may lead to complications if we're using `docker-compose run ...` in the `make.xsh` script to run this container for a manual backup/restore process. You could get around this in one of two ways:

1. replace `host.docker.internal` with `${POSTGRES_HOST?Variable not set}` and supply that environment variable from `make.xsh`
2. In `make.xsh` use `docker-compose run -e POSTGRES_HOST=<host>` where `<host>` is `localhost` if the host OS is linux and `host.docker.internal` if mac. FYI, there is a `get_localost()` function in `make.xsh` that already does this.

Reply:
I just finished making all of these changes